src/realm/query_expression.hpp

```diff
@@ -2620,6 +2795,18 @@ class Columns : public Subexpr2<T> {
     }
 };
 
+template <class T>
+void Columns<T>::evaluate(size_t index, ValueBase& destination)
```

Comment:
Btw. many (all?) functions in query_expression.hpp  are inlined. Have you any reflection on what that does to code size vs. speed?